src/Microsoft.Health.Dicom.Core/Features/Workitem/WorkitemQueryResponseBuilder.cs

```diff
@@ -7,19 +7,83 @@
 using System.Linq;
 using EnsureThat;
 using FellowOakDicom;
-using Microsoft.Health.Dicom.Core.Features.Query;
 using Microsoft.Health.Dicom.Core.Features.Query.Model;
 
 namespace Microsoft.Health.Dicom.Core.Features.Workitem
 {
     public static class WorkitemQueryResponseBuilder
     {
+        /// <summary>
+        /// Workitem attributes with a return key type of 1 or 2 (including conditionals).
+        /// <see href='https://dicom.nema.org/medical/dicom/current/output/html/part04.html#table_CC.2.5-3'/>.
+        /// </summary>
+        public static readonly HashSet<DicomTag> RequiredReturnTags = new HashSet<DicomTag>
+        {
+            // SOP Common Module
+            DicomTag.SpecificCharacterSet,
+            DicomTag.SOPClassUID,
+            DicomTag.SOPInstanceUID,
+
+            // Unified Procedure Step Scheduled Procedure Information Module
+            DicomTag.ScheduledProcedureStepPriority,
+            DicomTag.ProcedureStepLabel,
+            DicomTag.WorklistLabel,
+            DicomTag.ScheduledProcessingParametersSequence,
+            DicomTag.ScheduledStationNameCodeSequence,
+            DicomTag.ScheduledStationClassCodeSequence,
+            DicomTag.ScheduledStationGeographicLocationCodeSequence,
+            DicomTag.ScheduledHumanPerformersSequence,
+            DicomTag.ScheduledProcedureStepStartDateTime,
+            DicomTag.ScheduledWorkitemCodeSequence,
+            DicomTag.InputReadinessState,
+            DicomTag.InputInformationSequence,
+            DicomTag.StudyInstanceUID,
+
+            // Unified Procedure Step Relationship Module
+            DicomTag.PatientName,
+            DicomTag.PatientID,
+
+                // Issuer of Patient ID Macro
+                DicomTag.IssuerOfPatientID,
```

Comment:
nit: Spacing

Reply:
I felt indenting this subsection, while nonstandard, help to make the macro fields more readable and clarifies that they are not a separate module.